package.json

```diff
@@ -50,15 +56,12 @@
     "jsonwebtoken": "8.0.0",
     "lodash": "4.17.5",
     "method-override": "3.0.0",
+    "mongoose": "5.4.19",
     "node-pushnotifications": "^1.0.20",
     "node-rest-client": "3.1.0",
     "passport": "0.4.0",
     "passport-jwt": "4.0.0",
-    "pg": "6.1.6",
-    "pg-hstore": "2.3.2",
     "request": "~2.85.0",
-    "sequelize": "^4.28.6",
-    "sequelize-cli": "^4.1.1",
```

Comment:
Please move `sequelize`, `sequelize-cli`, `pg`, and `pg-hstore` back to `dependencies`